pkg/operator/operator.go

```diff
@@ -183,8 +194,9 @@ func (o *Operator) processNextWorkItem() bool {
 		return true
 	}
 
+	o.reconcileErrors.Inc()
```

Comment:
we should additionally have a counter to track all reconcile attempts, so we can alert on the error rate

Reply:
:tada: